src/tss2-esys/esys_int.h

```diff
@@ -84,6 +84,11 @@ typedef struct {
     TPM2B_AUTH authData;
 } HMAC_Start_IN;
 
+typedef struct {
+    TPM2B_AUTH *auth;
+    TPM2B_AUTH authData;
+} MAC_Start_IN;
+
```

Comment:
@williamcroberts I had to add this struct for the ESAPI MAC functions. This is not a part of the spec, but maybe is used internally. Please let me know if adding this is ok..

Reply:
Looks the same as HMAC_START_In, perhaps just add a typedef rather than a new definition. Otherwise makes sense to match up with new MAC code.

Reply:
@ahbishek typedef this?
`typedef HMAC_Start_IN MAC_Start_IN;`
over the explicit struct.